environment/deployments/panda/env/dev.tfvars

```diff
@@ -133,4 +133,5 @@ image         = "centos-7-v20210316"
 tags          = ["allow-ssh"]
 type          = "pd-standard"
 # source_image_family  = "centos-7"
-# source_image_project = "centos-cloud"
\ No newline at end of file
+# source_image_project = "centos-cloud"
```

Comment:
This change should probably be on a separate commit.

Reply:
I did this change only to trigger the GitHub Action pipeline